test/metabase/api/common/internal_test.clj

```diff
@@ -89,6 +90,19 @@
   (let [{:keys [state po-box archipelago]} body]
     (str/join " | " [state po-box archipelago])))
 
+(api/defendpoint GET "/with-query-params/"
```

Comment:
Here's a test that shows how to use query params (sort of a miss last time around).